src/test/java/net/snowflake/client/jdbc/FileUploaderSessionlessTest.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
+ */
+package net.snowflake.client.jdbc;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.*;
+import net.snowflake.client.jdbc.cloud.storage.StageInfo;
+import net.snowflake.common.core.RemoteStoreFileEncryptionMaterial;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/** Tests for SnowflakeFileTransferAgent.expandFileNames */
+public class FileUploaderSessionlessTest {
+  @Rule public TemporaryFolder folder = new TemporaryFolder();
+  private ObjectMapper mapper = new ObjectMapper();
+
+  private final String exampleJsonString =
```

Comment:
Having an example string is awesome - do we need all the attributes in it? Might make the tests more readable if it can be stripped down, but NBD either way.

Reply:
We can definitely strip it down.  I opted for the full thing to have the tests be slightly more realistic incase having extra information caused some kind of err (not sure how that would happen).  Happy to strip it down if you'd prefer the readability